src/Discord/Parts/User/Member.php

```diff
@@ -170,21 +170,32 @@ public function moveMember($channel)
      *
      * @param Role|int $role The role to add to the member.
      *
-     * @return bool Whether adding the role succeeded.
+     * @return bool returns whether role was added successfully
```

Comment:
Since this function performs an HTTP request it needs to return a promise:
`@return \React\Promise\Promise`